tests/acceptance/features/bootstrap/SettingsContext.php

```diff
@@ -390,10 +390,7 @@ public function sendRequestToSwitchSystemLanguage(string $user, string $language
 			],
 			JSON_THROW_ON_ERROR
 		);
-
-		$this->featureContext->setResponse(
-			$this->spacesContext->sendPostRequestToUrl($fullUrl, $user, $this->featureContext->getPasswordForUser($user), $body, $this->featureContext->getStepLineRef())
-		);
+		return $this->spacesContext->sendPostRequestToUrl($fullUrl, $user, $this->featureContext->getPasswordForUser($user), $body, $this->featureContext->getStepLineRef());
```

Comment:
would be easier to read if the linter allows
```suggestion
		return $this->spacesContext->sendPostRequestToUrl(
			$fullUrl,
			$user,
			$this->featureContext->getPasswordForUser($user),
			$body,
			$this->featureContext->getStepLineRef()
		);
```